packages/fuels/test/features/deploy.test.ts

```diff
@@ -180,39 +192,26 @@ describe('deploy', { timeout: 180000 }, () => {
     expect(firstFuelsContents.fooBar).toMatch(/0x/);
     expect(firstFuelsContents.upgradable).toMatch(/0x/);
 
-    /**
-     * a) Add helper
-     *   For interacting with deployed contract
-     */
-    async function executeTargetContract() {
-      const upgradableChunkedContractId = firstFuelsContents.upgradableChunked;
-      const upgradableChunkedAbi = JSON.parse(
-        readFileSync(
-          join(paths.upgradableChunkedContractPath, 'out', 'debug', 'upgradable-chunked-abi.json'),
-          'utf-8'
-        )
-      );
-
-      const targetContract = new Contract(
-        upgradableChunkedContractId,
-        upgradableChunkedAbi,
-        wallet
-      );
-
-      const res = await targetContract.functions.test_function().call();
-      const { value } = await res.waitForResult();
-
-      return value;
-    }
+    const contractId = firstFuelsContents.upgradableChunked;
+    const abi = JSON.parse(
+      readFileSync(
+        join(paths.upgradableChunkedContractPath, 'out', 'debug', 'upgradable-chunked-abi.json'),
```

Comment:
Will this always be debug?

Reply:
[As above](https://github.com/FuelLabs/fuels-ts/pull/3235#discussion_r1783016900)